mappings/net/minecraft/client/util/NetworkUtils.mapping

```diff
@@ -1,5 +1,10 @@
 CLASS net/minecraft/class_3521 net/minecraft/client/util/NetworkUtils
-	FIELD field_15664 downloadExecutor Lcom/google/common/util/concurrent/ListeningExecutorService;
+	COMMENT A few client-side networking utilities.
+	COMMENT
+	COMMENT @implNote This is not marked as client-only because it's used by the
+	COMMENT {@code /publish} command, which is only available to integrated servers
+	COMMENT yet was retained by proguard.
+	FIELD field_15664 DOWNLOAD_EXECUTOR Lcom/google/common/util/concurrent/ListeningExecutorService;
```

Comment:
```suggestion
	FIELD field_15664 EXECUTOR Lcom/google/common/util/concurrent/ListeningExecutorService;
```
In fact, this is used by client login network manager to query some login info in its `onHello` method.